From f1aef07dbac734640dae28e99512e5e3f4b9a957 Mon Sep 17 00:00:00 2001
From: psykose <alice@ayaya.dev>
Date: Sun, 5 Feb 2023 13:01:50 +0000
Subject: [PATCH] rtclock: fix implicit sleep declaration

otherwise clang16 fails to compile this code, and detects this as false
---
 cmake/rtclock/rtclock.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake/rtclock/rtclock.c b/cmake/rtclock/rtclock.c
index 851e78dc..439be41f 100644
--- a/cmake/rtclock/rtclock.c
+++ b/cmake/rtclock/rtclock.c
@@ -1,6 +1,7 @@
 #include <stdio.h>
 #include <time.h>
 #include <stdint.h>
+#include <unistd.h>
 int main( int argc, char** argv, char** env ) {
     uint64_t dt;
     struct timespec t0,t1;
-- 
2.39.1

